src/taxonomy/system-defined-badge/SystemDefinedBadge.scss

```diff
@@ -0,0 +1,7 @@
+.system-defined-badge {
+  font-size: 12px;
+}
+
+.system-defined-tooltip {
+  width: 300px;
+}
```

Comment:
Since this is defined twice, why not make it generic? i.e. `mw-300px` and use it in both places. 
The default is 480 I think.

Reply:
Updated https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/dab9d8a01a0389826e989252b112c3b70869e251